### DISCUSSION TITLE
How can I add release notes for firebase app distribution

### DISCUSSION THREAD

**Discussion by VadimZhuk0v** [codemagic-ci-cd/codemagic-docs#1189] [Q&A] [answered]
I want to add release notes with the list of all commits since the last release. How can I do this via codemagic?

**Comment by icarusdust** [comment, marked as answer]
Hi @VadimZhuk0v, if you have publishing section set for firebase app distribution, then release notes will be published there. Check out [this documentation](https://docs.codemagic.io/yaml-notification/publish-release-notes/).
